python/manylinux1/build_arrow.sh

```diff
@@ -97,36 +97,38 @@ echo "=== (${PYTHON_VERSION}) Building Arrow C++ libraries ==="
 ARROW_BUILD_DIR=/tmp/build-PY${PYTHON_VERSION}-${UNICODE_WIDTH}
 mkdir -p "${ARROW_BUILD_DIR}"
 pushd "${ARROW_BUILD_DIR}"
-cmake -DCMAKE_BUILD_TYPE=Release \
-    -DARROW_DEPENDENCY_SOURCE="SYSTEM" \
-    -DCMAKE_INSTALL_PREFIX=/arrow-dist \
-    -DCMAKE_INSTALL_LIBDIR=lib \
-    -DARROW_BUILD_TESTS=OFF \
-    -DARROW_BUILD_SHARED=ON \
+cmake \
+    -DCMAKE_BUILD_TYPE=Release \
     -DARROW_BOOST_USE_SHARED=ON \
+    -DARROW_BUILD_SHARED=ON \
```

Comment:
Not in the scope for this PR, but should we add `ARROW_BUILD_STATIC=OFF` here?

Reply:
Probably, created jira https://issues.apache.org/jira/browse/ARROW-7676